service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersMidEventControllerTest.java

```diff
@@ -83,15 +94,41 @@ void orderSelectionShouldPopulateQuestionConditionHolder() {
 
         AboutToStartOrSubmitCallbackResponse response = postMidEvent(caseData, "order-selection");
 
-        assertThat(response.getData().get("orderTempQuestions")).isEqualTo(
-            Map.of(
-                "approvalDate", "YES",
-                "approver", "YES",
-                "previewOrder", "YES",
-                "furtherDirections", "YES",
-                "whichChildren", "YES"
-            )
-        );
+        Map<String, String> expectedQuestions = new HashMap<>(getOrderQuestions());
+        expectedQuestions.putAll(Map.of(
+            "approvalDate", "YES",
+            "approvalDateTime", "NO",
+            "epoIncludePhrase", "NO",
+            "epoOrderType", "NO",
+            "epoChildrenDescription", "NO",
+            "epoExpiryDate", "NO",
+            "epoPreventRemoval", "NO"
+        ));
+
+        assertThat(response.getData().get("orderTempQuestions")).isEqualTo(expectedQuestions);
+    }
+
+    @Test
+    void epoOrderSelectionShouldPopulateQuestionConditionHolder() {
```

Comment:
Although I am not complaining about it do we need a test for the specific questions of the order type in the integration tests. Surely that would be covered in the unit tests and in the int test we just need to make sure that it has populated it correctly for one type of order to ensure that the data is actually returned